docs/docs/appendix.md

```diff
@@ -6,15 +6,15 @@ For each table containing an `#[auto_inc]` column, SpacetimeDB creates a sequenc
 
 ### How It Works
 
-* Sequences in SpacetimeDB use Rust’s `i128` integer type.
```

Comment:
If these docs tweaks are important to you, can you please move them into a separate PR? They seem entirely unrelated to the (meaningfully large) actual goal of this PR.

Reply:
I typically do something like:
```bash
git checkout -b new-branch origin/master
git checkout old-branch -- file1 file2 ...
git commit -am 'Moving files from other thing'
git push -u
gh pr create --web
```

Reply:
Yes, this is because I'm running the same pnpm format on all typescript directories (directories with package.json). I can exempt markdown from the formatting and add that in a separate PR if you'd like.

Reply:
I do think that eventually we should have docs under prettier so that we don't have non-substantive formatting changes to the docs causing diffs in the future.

Reply:
I've added `**/*.md` the docs `.prettierignore`